libs/components/layout/src/lib/modules/action-button/action-button-container.component.ts

```diff
@@ -74,6 +74,8 @@ export class SkyActionButtonContainerComponent
 
   private _themeName: string;
 
+  private _viewInitialized = false;
```

Comment:
The underscore is used for values associated with getters/setters. Maybe use `#viewInitialized` instead?